google-cloud-speech/google-cloud-speech.gemspec

```diff
@@ -12,6 +12,12 @@ Gem::Specification.new do |gem|
   gem.homepage      = "https://github.com/googleapis/googleapis"
   gem.license       = "Apache-2.0"
 
+  gem.post_install_message =
+    "This release, 0.30.0, introduces breaking changes relative to the "\
+    "previous release, 0.29.0. For more details and instructions to migrate "\
+    "your code, please visit the migration guide: "\
+    "https://cloud.google.com/speech/docs/ruby-client-migration."
```

Comment:
Not yet published but will be at: https://cloud.google.com/speech-to-text/docs/ruby-client-migration

Reply:
K, updated the URL. Thanks!